.github/workflows/conformance.yml

```diff
@@ -65,7 +65,8 @@ jobs:
           sudo mv kind /usr/local/bin
       - name: Create K8s cluster
         run: |
-          # The command also loads local antrea/antrea-ubuntu:latest into Nodes if it exists.
+          # The command also loads local antrea/antrea-agent-ubuntu:latest and antrea/antrea-controller-ubuntu:latest
```

Comment:
After split image changes, kind test bed (ci/kind/kind-setup.sh) will load antrea/antrea-controller-ubuntu:latest ( controller image) , IMO kind test bed shld use agent ubuntu image for worker node and controller image for control plane node,  description needs to be more specific -> https://github.com/antrea-io/antrea/pull/5794/files#diff-0cc131dea0ffb01ed0188ee08cd005d0e72e4064cfc945307df6a4a9cca68669R22-R23

Reply:
we have antrea-agent also in controlplane node.
When we apply antrea we get 3 antrea-agents and 1 antrea-controller, so there is one agent each on each of the nodes. We not only have antrea-controller in the control-plane node.